frontend/stubs/libraries/math.py

```diff
@@ -0,0 +1,431 @@
+from typing import Tuple, List
+
+e = 2.718281828459045
```

Comment:
I took it from PyCharm's stub files. I thought it was better than typeshed because it contained docstrings for all functions.